yarn.lock

```diff
@@ -4041,6 +4041,13 @@
   resolved "https://registry.yarnpkg.com/@types/json-schema/-/json-schema-7.0.3.tgz#bdfd69d61e464dcc81b25159c270d75a73c1a636"
   integrity sha512-Il2DtDVRGDcqjDtE+rF8iqg1CArehSK84HZJCT7AMITlyXRBpuPhqGLDQMowraqqu1coEaimg4ZOqggt6L6L+A==
 
+"@types/mdx-js__react@^1.5.1":
```

Comment:
We should have this package in `dependencies` of `@theme-ui/sidenav`, right?

Reply:
@hasparus please see here https://github.com/system-ui/theme-ui/pull/917/files#r426131822

Reply:
DevDep moved into package